tests/store/artifact/test_models_artifact_repo.py

```diff
@@ -3,86 +3,69 @@
 from unittest.mock import Mock
 
 from mlflow.entities.model_registry import ModelVersion
-from mlflow.exceptions import MlflowException
-from mlflow.store.artifact.databricks_model_artifact_repo import DatabricksModelArtifactRepository
+from mlflow.store.artifact.databricks_models_artifact_repo import DatabricksModelsArtifactRepository
 from mlflow.store.artifact.models_artifact_repo import ModelsArtifactRepository
 from mlflow.tracking import MlflowClient
 
-# pylint: disable=unused-import
-from tests.store.artifact.test_databricks_model_artifact_repo import host_creds_mock
-
-DATABRICKS_MODEL_ARTIFACT_REPOSITORY_PACKAGE = (
-    "mlflow.store.artifact.databricks_model_artifact_repo"
-)
-DATABRICKS_MODEL_ARTIFACT_REPOSITORY = (
-    DATABRICKS_MODEL_ARTIFACT_REPOSITORY_PACKAGE + ".DatabricksModelArtifactRepository"
-)
-
-
-@pytest.fixture
-def mock_get_model_version_download_uri(artifact_location):
```

Comment:
Initially this function was being used everywhere in the test, since we were getting the download uri and then choosing the repo to instantiate.
Since that logic changed, this mock was not as important, as it was needed in just 1-2 tests out of the 10.